client/src/components/FormArtistAdd/FormArtistAdd.js

```diff
@@ -0,0 +1,42 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import { Field, reduxForm } from 'redux-form';
+import { requiredValidate } from '../../utils/validation';
+import { artistAdd } from '../../redux/artist';
+
+import Button from '../Button';
+import Input from '../Input';
+
+class FormArtistAdd extends Component {
+  submit = (values, dispatch, props) => {
+    const { artistAdd, submitCallback } = this.props;
+    artistAdd(values, submitCallback);
+  };
+
+  render() {
+    const { props, submit } = this;
+    const { handleSubmit } = props;
+
+    return (
+      <form onSubmit={handleSubmit(submit)}>
+        <Field
+          component={Input}
+          label="Name"
+          name="name"
+          autoFocus
+          validate={requiredValidate}
+        />
+        <Button type="submit">Submit</Button>
+      </form>
```

Comment:
@dzimmerman902 I remember when you had the Edit Artist form laid out in the non-inline way, the form looked nicer. In this implementation, there's no vertical space between the `Field` and `Button` components. Do you remember if you had the form set up as `display:grid` in that implementation? Let me know and I'll get that implemented here!

Reply:
@seankwilliams  Correct, I did use display: grid and for the spacing, you can use grid-column-gap or grid-column-row for your needs. They are pretty much gutters between the elements.

Also, how do you do multi line comments?

Reply:
This is my go to for all css properties.

https://css-tricks.com/snippets/css/complete-guide-grid/

Reply:
@dzimmerman902 Thanks! I've pushed changes to set up `display:grid` styles for the form, and this is ready for your review now.

What are you referring to re:multi line comments?

Reply:
@seankwilliams 
<img width="666" alt="Screen Shot 2019-04-02 at 11 59 53 AM" src="https://user-images.githubusercontent.com/18602363/55425141-f3e95800-553e-11e9-82e1-75d6ab723cb0.png">
When I make a comment, it only does single lines. In this example, it shows 4 line

Reply:
@dzimmerman902 Oh! I have no idea actually lol. Maybe because I'm commenting on the final line of code changes? I try to do that so the comment doesn't break up the flow of the code but maybe showing more lines of the changes is an unintended side effect of that